PAN-OS - DNS Sinkhole (#27700)
demisto/content: Packs/PAN-OS/ReleaseNotes/2_0_0.md

#### Playbooks

##### New: PAN-OS - Configure DNS Sinkhole

> - New: This playbook creates a DNS sinkhole in PAN-OS Next Generation Firewalls.

Reviewer: ```suggestion
New: This playbook creates a DNS sinkhole in PAN-OS Next Generation Firewalls.
```

-
##### New: PAN-OS - Extract IPs From Traffic Logs To Sinkhole

> - New: This playbook searches PAN-OS traffic logs and returns the list of IP addresses that communicated with the PAN-OS sinkhole address.

Reviewer: ```suggestion
New: This playbook searches PAN-OS traffic logs and returns the list of IP addresses that communicated with the PAN-OS sinkhole address.
```

##### New: PAN-OS - Add Anti-Spyware Security Profile To Rule

> - New: Added a playbook that adds an Anti-Spyware Security Profile to a security rule in PAN-OS Next Generation Firewalls in a safe manner. (Available from Cortex XSOAR 6.8.0).

Reviewer: ```suggestion
New: This playbook adds an Anti-Spyware Security Profile to a security rule in PAN-OS Next Generation Firewalls in a safe manner. (Available from Cortex XSOAR 6.8.0).
```
